Makefile

```diff
@@ -93,3 +93,10 @@ clean-docker-images:
 		echo "<noop>";\
 	fi
 
+net:
+	tunctl -t tap0 -u `whoami`
+	brctl addif docker0 tap0
+	ip link set tap0 up
+
+run-debian:
```

Comment:
This target doesn't really mention `debian` anywhere in the command. Probably, `run-qemu-with-net` or something like that would be a more suitable name. Also, it de facto depends on the `linux.img` and `net` targets. I think you could just reflect this fact in the target line.

Reply:
replaced